pkg/cmd/server/start/start_node.go

```diff
@@ -280,8 +274,13 @@ func StartNode(nodeConfig configapi.NodeConfig) error {
 
 	config.EnsureVolumeDir()
 	config.EnsureDocker(docker.NewHelper())
+	plugin, endpointFilter := InitSDNPlugin(config, nodeConfig)
```

Comment:
> plugin and endpointFilter could be added to the *kubernetes.NodeConfig that BuildKubernetesNodeConfig returns

@liggitt I like that, but we'll need to find somewhere else for 'type FilteringEndpointsConfigHandler' to live because we have an import cycle.  openshift-sdn factory needs that type as the return value from NewPlugin() so it has to import origin/pkg/cmd/server/kubernetes but of course node_config.go is already in origin/pkg/cmd/server/kubernetes and needs to import factory.

Thoughts on where to put FilteringEndpointsConfigHandler then?  Maybe api/types.go?